### ISSUE TITLE
about the value of  γ

### ISSUE THREAD

**Issue by Cocoalate** [unique-chan/Complement-Cross-Entropy#3]
γ is set to -1 according to the paper, but in this code  γ = 5?
What value should γ be?
And what is the normal loss value of complement cross entropy? I tried in keras but ths loss is very high.

**Comment by unique-chan** [comment]
Sorry for the late reply. We recommend to use **gamma = -1** according to the paper. For this, all you need is to initialize the **self.balancing_factor as 1** (i.e. self.balancing_factor = 1).
